design-system/packages/fields/src/types.ts

```diff
@@ -1,3 +1,6 @@
 export type WidthType = 'small' | 'medium' | 'large' | 'full';
 export type SizeType = 'small' | 'medium' | 'large';
 export type ShapeType = 'square' | 'round';
+
+// A primitive data type for date fields, which is an un-formatted ISO8601 date string or null for empty
```

Comment:
this comment is wrong, no? Isn't empty represented by `""`? (I think null would be a better representation but don't mind strongly, just don't want misleading comments)

Reply:
good catch, its also no longer an unformatted ISO8601 string. Comment updated for parity